src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -29,4 +34,50 @@ public static String convertToJsonUsingGson(Object object) {
     Gson gson = new Gson();
     return gson.toJson(object);
   }
+
+  /*
```

Comment:
nit: Javadoc should start with a /**

```suggestion
  /**
```

Reply:
Done!